src/gs1/gs1-resolver.controller.ts

```
@@ -0,0 +1,185 @@
+  @Get('system/metadata')
+  async getSystemMetadata() {
+    return this.gs1Service.getSystemMetadata();
+  }
+
+  @Get('system/product-index')
+  async getProductIndex() {
+    return this.gs1Service.getProductIndex();
+  }
+
+  // Data Integrity Verification endpoints
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)